components/observations/ObservationDetailView.tsx

```diff
@@ -93,13 +95,44 @@ export const TableRow = ({label, value}: {label: string; value: string}) => (
   </HStack>
 );
 
+export const WeatherCard = ({observation, ...props}: {observation: Observation} & CardProps) => {
+  const {weather = {}, weather_summary = ''} = observation.advanced_fields || {};
+  // observation.advanced_fields.weather might be missing, or might be an object that's filled with empty strings -
+  // in either of those cases, we don't have weather data to render.
+  const hasWeatherEntries = Object.entries(weather || {}).some(([_k, v]) => Boolean(v));
+
+  if (!hasWeatherEntries && !weather_summary) {
+    return null;
+  }
+
+  const {cloud_cover, air_temp, recent_snowfall, rain_elevation, snow_avail_for_transport, wind_loading} = weather;
+
+  return (
+    <Card borderRadius={0} borderColor="white" header={<BodyBlack>Weather</BodyBlack>} {...props}>
+      <VStack space={8} width="100%">
+        {/* Using Boolean() here so that we don't end up rendering empty strings inline */}
+        {Boolean(weather_summary) && <HTML source={{html: weather_summary}} />}
+        {Boolean(cloud_cover) && <TableRow label={'Cloud Cover'} value={FormatCloudCover(cloud_cover as CloudCover)} />}
+        {Boolean(air_temp) && <TableRow label={'Temperature (F)'} value={air_temp || 'Unknown'} />}
```

Comment:
nit: do you think we can add something to `TableRow` that knows how to default? Perhaps a prop for the default value, which itself defaults to `'Unknown'` ?

Reply:
well for this particular line, `|| 'Unknown'` doesn't actually do anything! we've already checked that `air_temp` has a value, so we don't need a default. I'll just rip that out.